portfolio/src/main/java/com/google/sps/services/Services.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.services;
+/********************************************
+ * This class will host all utility methods
+ **/
+
+ import com.google.gson.Gson;
+
+ public class Services{
+
+   /**********************************************
+    * Converts a Java Object into JSON String
+    * @param : A Java Object
+    * @return : Json String 
+    **/
+    public static String toJson(Object o){
+        Gson gson = new Gson();
+        return gson.toJson(o);
```

Comment:
If you want to be more concise, 
```suggestion
        return new Gson().toJson(o);
```